auslib/test/test_db.py

```diff
@@ -2422,6 +2422,32 @@ def testGetRulesMatchingQueryChannelGlobbing(self):
         rules = self._stripNullColumns(rules)
         self.assertEquals(rules, expected)
 
+        # To ensure length of ruleChannel is >=3
+        expected = [dict(rule_id=2, priority=100, backgroundRate=100, channel='r*', update_type='z', data_version=1)]
```

Comment:
Please put each of these cases into their own method. When you include them in existing method, it means they won't run at all if an earlier assert fails.